### DISCUSSION TITLE
Time to start stop tracing

### DISCUSSION THREAD

**Discussion by TurboTronix** [blakeblackshear/frigate#883] [Ideas]
Is there a way to add a time when you want a camera to start/stop tracing?

For instance i have an indoor camera that i only want to trace during the night since during the day there way too much traffic.

**Comment by goulaobangzi** [comment]
+1 for hardware optimazation.

**Comment by paularmstrong** [comment]
You should use another system (like Home Assistant, Node-RED, or other automation platform) for this. You can hook into the MQTT server and start/stop detection that way: https://blakeblackshear.github.io/frigate/usage/mqtt

**Comment by goulaobangzi** [comment]
The main propose it's do reduce the hardware usage.

Mqtt can start and stop the detection but I believe it still takes the
resources to maintain the detention.

**Comment by TurboTronix** [comment]
Yeah the idea was to limit the usage of cpu when tracing is not needed...

I thought maybe have the tracing for a particular cam disabled or enabled based on hassio alarm but i don't know if you can control frigate from hassio if it's located on a diff pc.
